Microsoft.Toolkit.Uwp.UI.Controls/PullToRefreshListView/PullToRefreshListView.cs

```diff
@@ -56,6 +57,16 @@ public class PullToRefreshListView : ListView
         public static readonly DependencyProperty RefreshIndicatorContentProperty =
             DependencyProperty.Register(nameof(RefreshIndicatorContent), typeof(object), typeof(PullToRefreshListView), new PropertyMetadata(null));
 
+        #region IsPullToRefreshWithMouseEnabled
+
+        /// <summary>
+        /// IsPullToRefreshWithMouseEnabled Dependency Property
+        /// </summary>
+        public static readonly DependencyProperty IsPullToRefreshWithMouseEnabledProperty =
```

Comment:
Would it be better to have a property to enable the feature for all non-touch pointer types ?